superflore/utils.py

```diff
@@ -46,7 +46,7 @@ def get_pr_text(comment=None):
     msg = ''
     if comment:
         msg += '%s\n' % comment
-    msg += 'To reproduce this PR, run the following command.\n\n'
+    msg += 'This pull request was generated by issuing:\n\n'
```

Comment:
```suggestion
    msg += 'This pull request was generated by running the following command:\n\n'
```

Reply:
Just saw you had both files changed here after changing them. :) Thank you for that!